### DISCUSSION TITLE
网速问题

### DISCUSSION THREAD

**Discussion by xc-bnu** [getlantern/lantern-forum-cn#955] [Q&A]
@Mike777Khoo，你好，我从今年6月（续费两年）以来，一直到现在，能正常使用的时候非常少，基本上是处于只要连上lantern，网速就会非常慢，具体情况是：连google网页都打不开，其他的外网网页也是处于打不开状态。之前有反映过这样的问题，但是一直都没有得到解决。还是说你们那边根本就解决不了这个问题。花了那么多钱，并没有给我带来良好的体验效果。我很期待能及时得到你们那边非常有效的反馈以及有效的解决方案。如果解决不了，那只有申请退款了。不然每次想连lantern登外网网站，一直处于打不开状态一直刷新页面，给我整出抑郁症了，那损失就更严重了。
本人账号的邮箱是2491730877@qq.com，期待你们及时的反馈。

**Comment by MaxMKMax** [comment]
您好! 谢谢您通知我们. 

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by xc-bnu** [reply]
8位号码为：77985239

**Reply by Lewis0000221** [reply]
@Mike777Khoo 请问现在都退款申请都不再理会了吗？我申请退款的帖子都不回了吗？
